src/main/kotlin/model/Delius.kt

```diff
@@ -51,20 +52,32 @@ class Delius private constructor() {
         "Community API",
         "API over the nDelius DB used by HMPPS Digital team applications and services", "Java"
       ).apply {
-        APIDocs("https://community-api-public.test.delius.probation.hmpps.dsd.io/swagger-ui.html").addTo(this)
+        APIDocs("https://community-api-public.test.delius.probation.hmpps.dsd.io/swagger-ui/index.html").addTo(this)
         setUrl("https://github.com/ministryofjustice/community-api")
         uses(db, "connects to", "JDBC")
         ec2.add(this)
       }
 
+      offenderSearchIndexer = system.addContainer(
+        "Probation Offender Search Indexer",
+        "Service to update nDelius offender data held in Elasticsearch",
+        "Kotlin"
+      ).apply {
+        APIDocs("https://probation-search-indexer.hmpps.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config").addTo(this)
+        setUrl("https://github.com/ministryofjustice/probation-offender-search-indexer")
+        uses(elasticSearchStore, "Indexes offender data from nDelius to the Elasticsearch Index")
```

Comment:
🤔 Does this mean `elasticSearchStore` needs to have a connection to the nDelius database?

Reply:
Not quite, the "indexer" has a connection to the community-api that connects to database. And pushes data into Elastic Search.